### ISSUE TITLE
Disable dragging/swiping of card

### ISSUE THREAD

**Issue by missphyrgames** [jushutch/swiping_card_deck#35]
**Before posting your question** :heavy_check_mark:

- [x] I acknowledge that I've read the [Code of Conduct](https://github.com/jushutch/swiping_card_deck/blob/ab3b8f224ea8f9eda6a925d98752fa30ba1b386e/CODE_OF_CONDUCT.md)

**Question :grey_question:**

Hi, I was wondering how I can disable the dragging/swiping of the card by the user and only allow it to be interacted through buttons? I understand how to make it work with the buttons through the example code provided but not sure how to disable the swiping functionality.

**Comment by jushutch** [comment]
Hi @missphyrgames, thank you for your question! At this point, there is unfortunately no good way to disable the swiping feature while keeping the buttons functional. However, I'd be happy to add this feature as soon as possible and release it in v1.3.0! I'll update this issue once v1.3.0 is released and explain how to use the new feature 😄

**Comment by jushutch** [comment]
I've just released v1.3.0 which adds this functionality 🎉  

To prevent cards from being swiped by dragging, set the new `disableDragging` parameter to `true` when you create the `SwipingCardDeck` widget. For example:
```
SwipingCardDeck(
  cardDeck: getCardDeck(),
  onDeckEmpty: () => debugPrint("Card deck empty"),
  onLeftSwipe: (Card card) => debugPrint("Swiped left!"),
  onRightSwipe: (Card card) => debugPrint("Swiped right!"),
  cardWidth: 200,
  disableDragging: true,
);
```

Let me know if you need help or need additional functionality! Thank you again for raising this issue and contributing to open source projects!
